cli/service/service.go

```diff
@@ -76,6 +81,103 @@ func List() {
 	fmt.Printf("%s\n", jsonBytes)
 }
 
+func Log(ServiceName string, ServiceNameTail string) {
+	msgPrinter := i18n.GetMessagePrinter()
+	RefURL := ServiceName
```

Comment:
Upper case has special meaning in go, it is used for function names that are visible outside of a package, and it is used for constants. Upper case works fine, this is a convention, not a language rule. Variable names should begin with lower case.